### ISSUE TITLE
Create single-page website for Entrepreneurial Open Source podcast

### ISSUE THREAD

**Issue by shanda-eclipse** [EclipseFdn/entrepreneurialopensource.org#1]
We are looking to create a single-page website for a podcast we are launching in late April. 

Thabang has secured and transferred ownership the following domains:
- entrepreneurialopensource.com 
- entrepreneurialopensource.net
- entrepreneurialopensource.org

Mock-up prepared by @meKokabi : https://xd.adobe.com/view/346cf6b6-e24e-46ec-b817-9d2943befd95-613f/

Design assets: [assets (1).zip](https://github.com/EclipseFdn/entrepreneurialopensource.org/files/6243331/assets.1.zip)

We have two options for the embed code:

<div id='buzzsprout-small-player-1740291'></div><script type='text/javascript' charset='utf-8' src='https://www.buzzsprout.com/1740291.js?container_id=buzzsprout-small-player-1740291&player=small'></script>

OR

<div id='buzzsprout-large-player-1740291'></div><script type='text/javascript' charset='utf-8' src='https://www.buzzsprout.com/1740291.js?container_id=buzzsprout-large-player-1740291&player=large'></script>

We would like to see what they look like on the page, then decide which will be used. 

@KyleBrouseEclipse can provide support for content, links, etc.

**Comment by KyleBrouseEclipse** [comment]
For the "Suggest a guest/topic" button, it should be renamed to "Be A Guest" and link to this form: https://docs.google.com/forms/d/e/1FAIpQLSeM9Wh9KsvppT0IFvq5nONIFFXZOqgbNP_Pl__X-saDq4qYcQ/viewform?usp=sf_link

Thanks

**Comment by chrisguindon** [comment]
When would we like this landing page to be done?

**Comment by shanda-eclipse** [comment]
Jacob was working on the launch timeline. Ideally, we would have this live for April 26, when we start promoting the podcast, but we discussed and would still be OK as long as the site is launched by May 3.

**Comment by chrisguindon** [comment]
@shanda-eclipse is there a way for us to extract the text from the mockups? If not can you provide google docs, please?

It would also be useful to have the content for the metatags such as the title and the description that we should be using!

**Comment by chrisguindon** [comment]
The initial contribution has been made for the website: https://github.com/EclipseFdn/entrepreneurialopensource.org

The current state of this project can be viewed with this URL:
https://eclipsefdn-entrepreneurialopensource.netlify.app/

@shanda-eclipse What is the status of the domain name? Did we buy it? If so, I will work with @eclipsewebmaster to configure Netlify to start using it.

**Comment by shanda-eclipse** [comment]
Sorry, here is the link to the content. https://docs.google.com/document/d/1GmrJhPT9IEVQnJlLjdpjkRpe93ReG47PL6TVux4TKGk/edit?usp=sharing

**Comment by shanda-eclipse** [comment]
> @shanda-eclipse What is the status of the domain name? Did we buy it? If so, I will work with @eclipsewebmaster to configure Netlify to start using it.

Thabang sent the details to transfer the domains to webmaster and Sharon M on March 19.

**Comment by chrisguindon** [comment]
@shanda-eclipse I don't think there is enough time for us to build the website before April 26.

@linkfang This is your first project with a deadline May 3. This is going to be a tight deadline but @autumnfound and I can help you get this done in time!

@autumnfound I am adding you as an assignee since I expect you to be spending some time guiding @linkfang with this project!

**Comment by linkfang** [comment]
Hi @shanda-eclipse ,

For the social media buttons in breadcrumb (orange header bar) on Entrepreneurial Open Source page, what should they do? 

Should they lead the user to Eclipse social media page or share this page to user's personal social media? 
Can you please specify it? Thanks a lot!

**Comment by KyleBrouseEclipse** [comment]
Hi @linkfang , 

The social media buttons should be to share the page to the user's personal social media. 

As for the other "listen" links in the breadcrumb, they will link to the various platforms. Here is the Spotify link https://open.spotify.com/show/356sdz0jPAU7F8fYFpu0OX?si=KbFjypCrSuqH-sd7MvM2Fw. 

I am currently working on getting the podcast listed on other platforms, and will provide the links as they arise. 

Thanks!

**Comment by linkfang** [comment]
Hi @shanda-eclipse ,

Here is the preview URL for the Entrepreneurial Open Source page:
https://deploy-preview-2--eclipsefdn-entrepreneurialopensource.netlify.app/ 

And here are 3 things I would like to share:

1. RSS Feeds
For this preview version, we added both large podcast player and the small one. 
I personally prefer the small one a little more. Because we only have 1 episode for now, so the large one might look a little empty. I think we can use the small one for now and change to the large one when we have more episodes.
What do you think?

2. Meta tag
For the description meta tag for this page, I am using the first paragraph. I am not sure if this is good enough?

3. Listen buttons in social banner
So far, we got the Spotify link, so will add the other 3 when they are ready. 

Feel free to share your feedback! 
Thanks!

**Comment by shanda-eclipse** [comment]
Thanks Zhou. This looks great.

Some feedback:

- I think the paragraph "The Entrepreneurial Open Source Podcast features successful entrepreneurs..." needs to be a slightly bigger font. Maybe 20px @meKokabi can you recommend?
- Agree with the feedback on RSS feed display
> RSS Feeds
> For this preview version, we added both large podcast player and the small one.
> I personally prefer the small one a little more. Because we only have 1 episode for now, so the large one might look a little empty. I think we can use the small one for now and change to the large one when we have more episodes.
- Can we link the twitter handles @ gblondelle to https://twitter.com/gblondelle and @ t_mashologu to https://twitter.com/t_mashologu
- Above the Eclipse Foundation logo in the footer, can we add "Sponsored by:" in small text above

**Comment by meKokabi** [comment]
Thanks Zhou. it looks good. Agree with Shanda can you please increase the size of "The Entrepreneurial Open Source Podcast features successful entrepreneurs..." to 21 px. And also the banner image got cut please find attached image. Can you fix the banner's height please.
<img width="405" alt="Screen Shot 2021-04-27 at 3 25 18 PM" src="https://user-images.githubusercontent.com/75500459/116300741-3d7fed80-a76d-11eb-85ab-2240bef2b3cd.png">

**Comment by linkfang** [comment]
Hi @meKokabi ,

Would you like also to increase the font-size of the hosts description to 21px to make it consistent with the "The Entrepreneurial Open Source Podcast features successful entrepreneurs..." or just increase what you asked?

Thanks!

**Comment by meKokabi** [comment]
Hi, yes please. That would be more consistent.

**Comment by linkfang** [comment]
Hi @shanda-eclipse , @meKokabi ,

Changes have been made:
- Added the font size for description (to 21px).
- Kept the small player for RSS Feeds.
- Added the twitter link for 2 hosts.
- Added "Sponsored by:" above the white Eclipse logo on the bottom.
- Increased the banner height. (Note: The new height will look better for more cases, but when the browser window becomes very wide, the image will likely be cut a little to fit in the size, or the banner will be too big/high.)

Here is the preview link:
https://deploy-preview-2--eclipsefdn-entrepreneurialopensource.netlify.app/

Let me know if you have any other feedback on the new version.
Thanks!

**Comment by shanda-eclipse** [comment]
One final edit: can we remove the title "RSS Feeds". Going right from the description paragraph to the feed should be sufficient.
 
I'm going to share with Thabang and Gael for their feedback.

**Comment by linkfang** [comment]
> One final edit: can we remove the title "RSS Feeds". Going right from the description paragraph to the feed should be sufficient.

Yeah, it does look nice and right after removing it.
Preview link:
https://deploy-preview-2--eclipsefdn-entrepreneurialopensource.netlify.app/ 

> I'm going to share with Thabang and Gael for their feedback.

OK, sounds good! Thanks @shanda-eclipse !

**Comment by chrisguindon** [comment]
The LinkedIn link is missing in the footer. This might be a bug in our base theme.

I would not expect to see a Linkedin link if a URL was not set for it!

**Comment by autumnfound** [comment]
> > One final edit: can we remove the title "RSS Feeds". Going right from the description paragraph to the feed should be sufficient.
> 
> Yeah, it does look nice and right after removing it.
> Preview link:
> https://deploy-preview-2--eclipsefdn-entrepreneurialopensource.netlify.app/

I would put a margin-top: 40px on the rss-feeds class, give the feed a little room to breathe :)

**Comment by linkfang** [comment]
Hi @chrisguindon , yep, good catch! 
I think it's a bug in the base theme. The Eclipse IoT website has the same issue. https://iot.eclipse.org/ 
For now, I added a script to change the link of LinkedIn social button so that it could work properly.

Hi @autumnfound , yes, I agree. I put the margin-top as 35px to make is consistent with the margin-bottom of header "Entrepreneurial Open Source Podcast".

**Comment by shanda-eclipse** [comment]
FYI - @meKokabi is working with Thabang on some edits to this page. This will push out our launch of this page to next week (so don't feel rushed to get it live by EOD today).

**Comment by chrisguindon** [comment]
@shanda-eclipse,

When should we expect these changes?

**Comment by linkfang** [comment]
Hi @shanda-eclipse , @KyleBrouseEclipse 
Are the 3 listen links ready now? We got the Spotify one, but still need the Apple, Google and Youtube ones.
Could you share them to us when they are ready?

Thanks!

**Comment by shanda-eclipse** [comment]
Our team has made some modifications to the design:
https://xd.adobe.com/view/346cf6b6-e24e-46ec-b817-9d2943befd95-613f/

- Increase the size of the logo and "Be a guest" button
- Change the title capitalization to "Entrepreneurs at the intersection of open source and business"
- We've added the "Sponsored by" block towards the top, with the "Become a sponsor" button that links to https://outreach.eclipse.foundation/eospod
- Add back the title above the RSS feed as "Episodes"
- Swap out Thabang's headshot to the one provided

@KyleBrouseEclipse will provide the missing links tomorrow (he's off today)
@meKokabi will provide any updated assets

Thanks for your patience with this!

**Comment by meKokabi** [comment]
Please find the page screenshot  <img width="466" alt="Screen Shot 2021-05-04 at 3 23 45 AM" src="https://user-images.githubusercontent.com/75500459/117018168-43705400-acc2-11eb-8c0b-5c3c4c5467d1.png">

**Comment by meKokabi** [comment]
please consider the height of the hero banner image. I understand image goes inside the wrapper and page has fluid layout but please add some CSS rules to maintain the size/height. Of course you know better than me. I checked the source file and saw <meta name=viewport content="width=device-width,initial-scale=1"> which is great, so may be adding some inline rules to maintain the height.

.hero-image {
 background: url('my-mobile-hero-image.jpg') no-repeat center top;
 background-size: cover; 
height: 100vh;
}

By all means feel free to disregard this if you think it is not correct. You are the pro!

This is an screenshot of the first fold for your ref.
<img width="507" alt="Screen Shot 2021-05-04 at 10 53 03 AM" src="https://user-images.githubusercontent.com/75500459/117023273-f347c080-acc6-11eb-928f-9ed6209deb07.png">

**Comment by meKokabi** [comment]
Additional information :The size of the logo is 676 in 231 px on desktop view.
and here is an updated image for Thabang's headshot
![Thabang](https://user-images.githubusercontent.com/75500459/117024010-9e587a00-acc7-11eb-8f45-859df37422f5.png)

Thanks so kindly for your support and time. Please let me know if you need any additional assets/ information.

**Comment by autumnfound** [comment]
> please consider the height of the hero banner image. I understand image goes inside the wrapper and page has fluid layout but please add some CSS rules to maintain the size/height. Of course you know better than me. I checked the source file and saw which is great, so may be adding some inline rules to maintain the height.
> 
> .hero-image {
> background: url('my-mobile-hero-image.jpg') no-repeat center top;
> background-size: cover;
> height: 100vh;
> }
> 
> By all means feel free to disregard this if you think it is not correct. You are the pro!
> 
> This is an screenshot of the first fold for your ref.
> <img alt="Screen Shot 2021-05-04 at 10 53 03 AM" width="507" src="https://user-images.githubusercontent.com/75500459/117023273-f347c080-acc6-11eb-928f-9ed6209deb07.png">

With a few changes, that's a great suggestion, and we should totally look at adding a % height to give the general look and feel that is the target.

Technical details on implementation follows, feel free to skip this section! Using 100vh would cause the section to be the height of your browser (as vh indicates a percentage height of your viewport or browser, with a value of 1-100 being the % value), and is much too large a number for this section. We may want to consider 66vh if the general proportion is wanted at typical views. Additionally, we shouldn't use a hard `height` property, but instead use `min-height` in case someone uses a landscape device which would be...not great.

Height:
![image](https://user-images.githubusercontent.com/23018241/117033969-ddd79400-acd0-11eb-8e0c-fceda91f7d3b.png)

Min-height:
![image](https://user-images.githubusercontent.com/23018241/117034087-f6e04500-acd0-11eb-9215-5142a2203eff.png)

More normal aspect ratio (typical view), with min-height:
![image](https://user-images.githubusercontent.com/23018241/117034383-4161c180-acd1-11eb-986a-8c90992fe8c7.png)



> Additional information :The size of the logo is 676 in 231 px on desktop view.
> and here is an updated image for Thabang's headshot
> ![Thabang](https://user-images.githubusercontent.com/75500459/117024010-9e587a00-acc7-11eb-8f45-859df37422f5.png)
> 
> Thanks so kindly for your support and time. Please let me know if you need any additional assets/ information.

For images like thumbnails, it's best to provide images without the circular cutout (so just in the typical square image view). This allows the browser to instead just hide the parts of the image we don't want to see, and provides a better experience with viewing in some cases!

**Comment by meKokabi** [comment]
Thanks for information. Here You are! <img width="1037" alt="Thabang" src="https://user-images.githubusercontent.com/75500459/117048112-a375f300-ace0-11eb-9eb4-f6a572acf242.png">

**Comment by linkfang** [comment]
Changes have been made based on the new design.
Preview link: https://deploy-preview-2--eclipsefdn-entrepreneurialopensource.netlify.app/ 
Thanks!

**Comment by meKokabi** [comment]
Thanks Zhou, it looks good, just one more quick request, can you please fix
the padding-top and Padding-bottom of the Episodes' section ( same padding
please top and bottom). Other than that everything looks good from my side.
Thanks again for your help.

**Comment by linkfang** [comment]
Hi @meKokabi ,

OK, the margin of Episodes' section is fixed. Now both bottom and top are 60px.
Preview link:
https://deploy-preview-2--eclipsefdn-entrepreneurialopensource.netlify.app/

Just would like to explain why we didn't fix the height of image to be 100% of the banner container.

For the height of the image, I made changes based on Martin's advice. Now it will fit in more devices/sizes.
But for making the page responsive, there will be some situations that we need to cut off the height or width of the image to make sure it can fill/cover the banner container. Like this:
![cut-the-height](https://user-images.githubusercontent.com/43075439/117059510-e12d4880-aced-11eb-9e6d-016fd365f381.jpg)
or
![cut-the-width](https://user-images.githubusercontent.com/43075439/117059534-e68a9300-aced-11eb-9c54-14101cf875d6.jpg)

If we stick the height with 100% height of the banner container, then when the browser becomes wide enough (the width/height ratio is greater than the ratio of the image) the banner part will look like this:
![empty-width](https://user-images.githubusercontent.com/43075439/117059756-20f43000-acee-11eb-8098-b78c3cde45f8.jpg)

Neither using a background color to fill the empty width area or repeating the image is ideal for this situation. And we would also avoid making the height too big, in case users have to scroll down to finish reading/viewing the banner image.

And good to hear you feel good for the new preview site! :)
Thanks!

**Comment by meKokabi** [comment]
Thanks for elaboration, it looks great!

**Comment by linkfang** [comment]
Hi @KyleBrouseEclipse 
Just to confirm that if you would provide the missing links (listening links) today or tomorrow?
Thanks!

**Comment by chrisguindon** [comment]
> Hi @KyleBrouseEclipse
> Just to confirm that if you would provide the missing links (listening links) today or tomorrow?
> Thanks!

@shanda-eclipse,

How about we comment out this section while you guys work on getting them for us? Or would you prefer we postpone the launch until these links are ready?

**Comment by shanda-eclipse** [comment]
> How about we comment out this section while you guys work on getting them for us?

+1

Another edit: It looks like the RSS feeds are now narrower (same width as the paragraph above). Can they span the same width as the paragraph plus the sponsor box. 

![image](https://user-images.githubusercontent.com/61751191/117163757-f8be0d00-ad91-11eb-8620-3db26e566674.png)

**Comment by linkfang** [comment]
Hi @shanda-eclipse ,
Yeah, of course. Actually I made it narrower on purpose, because it looks this way in the XD design page so I thought that's something we would like to have.
So just to confirm that making it the same width as the paragraph above is what we need, right?

**Comment by shanda-eclipse** [comment]
> So just to confirm that making it the same width as the paragraph above is what we need, right?

Yes please

**Comment by linkfang** [comment]
> Yes please

OK, done and also hide the listen link section for now

Preview link:
https://deploy-preview-2--eclipsefdn-entrepreneurialopensource.netlify.app/

**Comment by shanda-eclipse** [comment]
Are we able to keep the spotify link since it was working? Or is it all or nothing?

**Comment by linkfang** [comment]
Sure, we can keep it and it's back.
https://deploy-preview-2--eclipsefdn-entrepreneurialopensource.netlify.app/

**Comment by shanda-eclipse** [comment]
+1 to go live.

@KyleBrouseEclipse can provide the missing links when he is in tomorrow

**Comment by KyleBrouseEclipse** [comment]
Google Podcast Link: https://podcasts.google.com/feed/aHR0cHM6Ly9mZWVkcy5idXp6c3Byb3V0LmNvbS8xNzQwMjkxLnJzcw

**Comment by KyleBrouseEclipse** [comment]
The Apple podcast platform has recently gone through an update and is very buggy. I've sent in a support request to get a working link.

**Comment by KyleBrouseEclipse** [comment]
Apple Podcast link is finally ready: https://podcasts.apple.com/us/podcast/entrepreneurial-open-source/id1565557488

**Comment by linkfang** [comment]
Apple Podcast link is added and now live.
https://entrepreneurialopensource.org/

Will reopen the issue once the YouTube one is ready
